src/components/Graph.react.js

```diff
@@ -279,6 +298,16 @@ const graphPropTypes = {
      */
     relayoutData: PropTypes.object,
 
+    /**
+     * Data that should be appended to existing traces. Has the form
+     * `[updateData, traceIndices]`, where `updateData` is an object
+     * containing the data to extend, and `traceIndices` is an array
+     * of trace indices that should be extended.
+     * Reference the Plotly.extendTraces API for full usage:
+     * https://plot.ly/javascript/plotlyjs-function-reference/#plotlyextendtraces
```

Comment:
Oh interesting - those docs don't include the last (optional) argument to `extendTraces`: `maxPoints`, which will truncate the beginning of the trace if there are more than that many points after extending. I've made a new issue to fix the docs https://github.com/plotly/documentation/issues/1284 but can we add it here?

https://github.com/plotly/plotly.js/blob/c60bac2c74ba73733e94a0d288bda481906b2b67/src/plot_api/plot_api.js#L978-L995